app/workers/rapid_ready_for_decision/form526_asthma_job.rb

```diff
@@ -9,27 +9,11 @@ class Form526AsthmaJob < Form526BaseJob
     sidekiq_options retry: 2
 
     def assess_data(form526_submission)
-      client = lighthouse_client(form526_submission)
-      response = client.list_resource('medication_requests')
-      med_request_data = RapidReadyForDecision::LighthouseMedicationRequestData.new(response)
-
-      body = <<~BODY
-        A single-issue asthma claim for increase was detected and offramped.<br/>
-        Submission ID: #{form526_submission.id}<br/>
-        Veterans Health API returned #{med_request_data.count} medication requests.<br/>
-        <br/>
-        Medications by year and status:<br/>
-        #{meds_by_year_and_status_as_string(med_request_data)}<br/>
-      BODY
-      ActionMailer::Base.mail(
-        from: ApplicationMailer.default[:from],
-        to: Settings.rrd.event_tracking.recipients,
-        subject: 'RRD claim - Offramped',
-        body: body
-      ).deliver_now
-
-      # returning nil will short-circuit the PDF generation step
-      nil
+      assessed_data = query_and_assess_lighthouse(form526_submission)
```

Comment:
All the query/assess code in this class are heavily modeled after the analogous code in the hypertension job. It's so similar that some day soon we'll probably want to refactor it into the base class.

Reply:
I'm realizing I'll need to flag certain medications for being "asthma-like" based on provided keywords (that we could store in constants). I wonder if that munging exercise should go in here somewhere.

I have an alternative way to do this in the pdf generator but seems off to me...